.github/workflows/transaction-exclusion-api-testing.yml

```
@@ -0,0 +1,51 @@
+name: transaction-exclusion-api-testing
+
+on:
+  workflow_call:
+    secrets:
+      DOCKERHUB_USERNAME:
+        required: true
+      DOCKERHUB_TOKEN:
+        required: true
+  workflow_dispatch:
+    inputs:
+      coverage:
+        description: To generate test report
+        required: false
+        type: boolean
+        default: false
+
+jobs:
+  run-tests:
+    runs-on: ubuntu-latest
```

```suggestion
    runs-on: ubuntu-24.04
```

we should use fixed versions. Otherise, when GitHub updates the `lastest` we may have issues.